src/pages/GetStarted/CustomWorkspaceTab/index.tsx

```diff
@@ -134,20 +115,17 @@ export class CustomWorkspaceTab extends React.PureComponent<Props, State> {
     }
   }
 
-  private getStorageType(devfile: che.WorkspaceDevfile): StorageType {
-    let storageType: StorageType;
-    // storage type
+  private getStorageType(devfile: che.WorkspaceDevfile): che.WorkspaceStorageType {
     if (devfile.attributes && devfile.attributes.persistVolumes === 'false') {
       const isAsync = devfile.attributes && devfile.attributes.asyncPersist === 'true';
       if (isAsync) {
-        storageType = StorageType.async;
+        return 'async';
```

Comment:
Why do you remove the common enum?

Reply:
It's because that enum made storage type and storage type title interchangeable. While adding the unit test I found out a bug that was caused by the reason I mentioned. 

I declared a type [`WorkspaceStorageType` ](https://github.com/che-incubator/che-dashboard-next/pull/83/files#diff-01fea99fd559d27f8e88980d5a2eef372b2833d7d8477d8ba32b7d66b687cc7fR32), added a service [`StorageTypes`](https://github.com/che-incubator/che-dashboard-next/pull/83/files#diff-38bd15564c43dc7d2b4a8264a3d3e94290a7db20d6ca6a839d7ff8215817e27b) and refactored related code. And that made things clear and explicit.

Reply:
As I understand correctly, it was several ways more to fix this problem:
1. Chang the approach for comparing
`StorageType[type] === StorageType.persistent`
 =>
`type => type ===  StorageType[StorageType.persistent]`
(Im I right?)
2. Create a  moc for this enum(for jest).

As for me, enum is much better than write
'persistent',  'persistent'  and 'async' dozens of times.

Reply:
It's not about comparing approach. There are several places where you don't really know whether `type` is a storage type or UD representation of storage type.

Reply:
discussed

Let it be.